mod-data-export/src/test/resources/domain/dataexport/features/file-definition.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Tests for file definitions
+
+  Background:
+    * url baseUrl
+
+    * callonce login testAdmin
+    * def okapiAdminToken = okapitoken
+
+    * callonce login testUser
+    * def okapiUserToken = okapitoken
+
+    # load variables
+    * callonce variables
```

Comment:
Does it make sense to make the step name more descriptive and remove the comment? e.g. `load callonce variables`

Reply:
Function renamed to "loadTestVariables"